app/core/container.py

```diff
@@ -170,7 +173,11 @@ def get_container_id(container_name: str) -> Optional[str]:
 
 
 def build_container(
-    container_name: str, volume_list, image_name: str, cpu: str
+    container_name: str,
+    volume_list,
+    image_name: str,
+    cpu: str,
+    container_config_dict: Union[Dict[Any, Any], None] = None,
```

Comment:
Can use Optional[Dict] as Optional is an alias for union with none

Reply:
done